tests/mail/test_sendtestemail.py

```diff
@@ -2,32 +2,75 @@
 
 from django.core import mail
 from django.core.management import call_command
-from django.test import SimpleTestCase
+from django.test import SimpleTestCase, override_settings
 
 
+@override_settings(
+    ADMINS=(('Admin', 'admin@example.com'), ('Admin and Manager (Admin)', 'admin_and_manager@example.com')),
+    MANAGERS=(('Manager', 'manager@example.com'), ('Admin and Manager (Manager)', 'admin_and_manager@example.com')),
+)
 class SendTestEmailManagementCommand(SimpleTestCase):
     """
     Test the sending of a test email using the `sendtestemail` command.
     """
 
-    def test_send_test_email(self):
+    def test_single_receiver(self):
         """
         The mail is sent with the correct subject and recipient.
         """
-        recipient = "joe@example.com"
-        call_command("sendtestemail", recipient)
+        recipient = 'joe@example.com'
+        call_command('sendtestemail', recipient)
         self.assertEqual(len(mail.outbox), 1)
         mail_message = mail.outbox[0]
         self.assertEqual(mail_message.subject[0:15], 'Test email from')
         self.assertEqual(mail_message.recipients(), [recipient])
 
-    def test_send_test_email_with_multiple_addresses(self):
+    def test_multiple_receivers(self):
         """
         The mail may be sent with multiple recipients.
         """
-        recipients = ["joe@example.com", "jane@example.com"]
-        call_command("sendtestemail", recipients[0], recipients[1])
+        recipients = ['joe@example.com', 'jane@example.com']
+        call_command('sendtestemail', recipients[0], recipients[1])
         self.assertEqual(len(mail.outbox), 1)
         mail_message = mail.outbox[0]
         self.assertEqual(mail_message.subject[0:15], 'Test email from')
-        self.assertEqual(mail_message.recipients(), recipients)
+        self.assertEqual(len(mail_message.recipients()), 2)
+        self.assertIn('joe@example.com', mail_message.recipients())
+        self.assertIn('jane@example.com', mail_message.recipients())
+
+    def test_manager_receivers(self):
+        """
+        The mail should be sent to the email addresses specified in
+        settings.MANAGERS.
+        """
+        call_command('sendtestemail', '--managers')
+        self.assertEqual(len(mail.outbox), 1)
+        mail_message = mail.outbox[0]
+        self.assertEqual(len(mail_message.recipients()), 2)
+        self.assertIn('manager@example.com', mail_message.recipients())
```

Comment:
is there a reason not to check list equality? use `sorted()` if ordering is non-deterministic. In that case we can remove the length assertion as well.